### PR TITLE
Resizable panels

### PR DESCRIPTION
**Pull request by bvaughn**
Installs [`react-resizable-panels`](https://www.npmjs.com/package/react-resizable-panels) package and uses it to replace all of the places we were using `SplitBox`.

Easiest to [review with whitespace changes hidden](https://github.com/replayio/devtools/pull/8401/files?w=1).

- [x] Update various resizable surface to use `react-resizable-panels` instead of `SplitBox`:
  - [x] Architecture test shell panels
  - [x] Secondary toolbox:
    - [x] Canvas / secondary toolbar split
    - [x] Secondary toolbox sub-panels:
      - [x] Console filters / messages
      - [x] Elements panel markdown tree and properties
      - [x] Network panel
  - [x] (Primary) toolbox and Source viewer
  - [x] (Secondary) toolbox and Source viewer
- [x] Delete old `SplitBox` component
- [x] Tests
  - [x] Update e2e tests to work with new markup
  - [x] Update screenshot tests to work with new markup
  
Note the Playwright tests pass **locally** for me but break in CI (on this PR and on `main`). I think this is related to a change @jasonLaster is currently rolling out and not caused by this PR.

https://user-images.githubusercontent.com/29597/209394310-71ba0bf9-ac14-451c-b948-1b920a7a0f5d.mp4

### CODE REVIEW

File: packages/bvaughn-architecture-demo/components/console/ConsoleRoot.tsx [hunk 1]
```diff
@@ -135,62 +136,124 @@ function Console({
...
+              </div>
+            </ErrorBoundary>
+          </Suspense>
+        </Panel>
+      </PanelGroup>
+    </div>
+  );
+}
+
+function ConsoleActionsRow({
```
Comment (jasonLaster) [inline packages/bvaughn-architecture-demo/components/console/ConsoleRoot.tsx:216]: would probably move this to its own module

**Review by jasonLaster — APPROVED** [review]
Code looks good. 

The one UX thing is that I could not find the drag handle between the video and the secondary panel.

**Comment by bvaughn** [comment]
> The one UX thing is that I could not find the drag handle between the video and the secondary panel.

Thanks for spotting this! I had the wrong Tailwind classes on it.

**Comment by bvaughn** [comment]
Going to wait and try landing this tomorrow since it's so close to the end of the day. Got a little held up tracking down unexpected screenshot differences (I think fixed by 58c9d301b2eb97ebb250ec6469e29033b0a47a7b).

**Pull request merged**